backend/djangoindia/db/models/event.py

```diff
@@ -89,3 +89,42 @@ def __str__(self) -> str:
         return (
             f"{self.first_name} {self.last_name} ({self.email}) --- {self.event.name}"
         )
+
+    
+
+class SponsorData(BaseModel):
+    SPONSOR_TYPE_CHOICES = [
+        ('individual', 'Individual'),
+        ('organization', 'Organization'),
+    ]
+
+    name = models.CharField(max_length=255)
+    contact = models.CharField(max_length=100)
+    sponsor_type = models.CharField(max_length=20, choices=SPONSOR_TYPE_CHOICES)
+    logo = models.ImageField(upload_to='sponsors/logos/')
+    url = models.URLField(max_length=200, blank=True, null=True)

```

Comment:
to be on a safer side.
```suggestion
    url = models.URLField(max_length=500, blank=True, null=True)

```